espnet2/bin/svs_inference.py

```diff
@@ -0,0 +1,532 @@
+#!/usr/bin/env python3
+
+"""Script to run the inference of singing-voice-synthesis model."""
+
+import argparse
+import logging
+from pathlib import Path
+import shutil
+import sys
+import time
+from typing import Any, Dict, Optional, Sequence, Tuple, Union
+
+import matplotlib
+import numpy as np
+import soundfile as sf
+import torch
+from typeguard import check_argument_types
+
+from espnet.utils.cli_utils import get_commandline_args
+from espnet2.fileio.npy_scp import NpyScpWriter
+from espnet2.tasks.svs import SVSTask
+from espnet2.torch_utils.device_funcs import to_device
+from espnet2.torch_utils.set_all_random_seed import set_all_random_seed
+
+from espnet2.svs.naive_rnn.naive_rnn import NaiveRNN
+from espnet2.svs.naive_rnn.naive_rnn_dp import NaiveRNNDP
+
+# from espnet2.svs.glu_transformer.glu_transformer import GLU_Transformer
+# from espnet2.svs.xiaoice.XiaoiceSing import XiaoiceSing
+# from espnet2.svs.xiaoice.XiaoiceSing import XiaoiceSing_noDP
+
```

Comment:
Maybe, we can remove these lines?

Reply:
Sure. These are models to be added in future PRs. Will remove it for now

Reply:
If there are some intentions, you can keep these lines.